src/rethinkdb-orm/error.cr

```diff
@@ -26,6 +26,10 @@ class RethinkORM::Error < Exception
         }
       end
 
+      if message.nil?
```

Comment:
You have set a default value for message as

```
message  = "Document has invalid fields"
```

Then this will never be nil

Reply:
woops, missed that